apps/system/lockscreen/js/lockscreen_charging.js

```diff
@@ -66,8 +69,8 @@ LockScreenChargingStatus.prototype.refresh = function cs_refresh() {
     }
   }
 
-  navigator.mozL10n.setAttributes(this.elements.charging,
-    l10nAttrs.id, l10nAttrs.args);
+  this.elements.charging.firstChild.data =
```

Comment:
Since after 3 months without comments people may know nothing about the reason why we do this.